code/code.py

```diff
@@ -79,7 +84,52 @@ def command_add(message):
         markup.add(c)
     msg = bot.reply_to(message, 'Select Category', reply_markup=markup)
     bot.register_next_step_handler(msg, post_category_selection)
+	
+	
+@bot.message_handler(commands=['search'])
+def command_select(message):
+    chat_id = message.chat.id
+   
+    message = bot.send_message(chat_id, 'what is the product Name?')
+    bot.register_next_step_handler(message, product_table)
+    
 
+def product_table(message):
+    try:
+        chat_id = message.chat.id
+        product_name = message.text
+        bot.send_message(chat_id, 'Loading....')
+        bot.send_message(chat_id, 'Compared prices are')
+        parser = argparse.ArgumentParser(description="Slash")
+        parser.add_argument('--search', type=str, help='Product search query')
+        parser.add_argument('--num', type=int, help="Maximum number of records", default=3)
+        parser.add_argument('--sort', type=str, nargs='+', help="Sort according to re (relevance: default), pr (price) or ra (rating)", default="re")
+        parser.add_argument('--link', action='store_true', help="Show links in the table")
+        parser.add_argument('--des', action='store_true', help="Sort in descending (non-increasing) order")
+        args = parser.parse_args()
+                
+        products1 = Scraped_data.searchAmazon(product_name)
+        products2 = Scraped_data.searchWalmart(product_name)
+        print(products1)
+
+        for sortBy in args.sort:
+            products1 = formatter.sortList(products1, sortBy, args.des)[:args.num]
+            products2 = formatter.sortList(products2, sortBy, args.des)[:args.num]
+            results = products1 + products2
+            results = formatter.sortList(results, sortBy, args.des)
+            
+
+        print()
```

Comment:
if you just require new lines then you can use \n before and after the string print statement you have in between to reduce the empty print calls made here.